tests/utils/utils.test.jsx

```diff
@@ -0,0 +1,296 @@
+import * as Utils from 'utils/utils.jsx';
+
+describe('Utils.isValidPassword', function() {
+    test('Password min/max length enforced if no EE password requirements set', function() {
+        for (const data of [
+            {
+                password: 'four',
+                config: { // not EE, so password just has to be min < length < max
+                  isEnterprise: false,
+                  isLicensed: true,
+                  isPasswordRequirements: true,
+                },
+                valid: false,
+            },
+            {
+                password: 'thistestpasswordismorethansixtyfourcharacterslongsoitstoolongtobeapassword',
+                config: { // not EE, so password just has to be min < length < max
+                    isEnterprise: false,
+                    isLicensed: true,
+                    isPasswordRequirements: true,
+                },
+                valid: false,
+            },
+            {
+                password: 'thisisavalidpassword',
+                config: { // not EE, so password just has to be min < length < max
+                    isEnterprise: false,
+                    isLicensed: true,
+                    isPasswordRequirements: true,
+                },
+                valid: true,
+            },
+            {
+                password: 'four',
+                config: { // not EE, so password just has to be min < length < max
+                    isEnterprise: true,
```

Comment:
Should this be EE?

Reply:
Yes. This suite turns one of the three booleans off for each set of three cases. So in the first three tests, `isEnterprise` is false, but in this set of three tests, `isLicensed` is false. In the last set of three tests, `isPasswordRequirements` is false. All three flags must be true for the EE passwords requirements to be checked, so this is the only way to check that the requirements are bypassed in all cases that they should be.

Reply:
Ah okay. I mean, the comment `not EE` might need to remove?